dev-docs/bidders/programmatica.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: bidder
+title: Programmatica
+description: programmatica bid adapter
+pbjs: true
+biddercode: programmatica
+media_types: banner, video
```

Comment:
Please add more meta data so publisher and ad tech vendors can vet your adapter properly.

- safeframes_ok: true/false
- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- ortb_blocking_supported: true/partial/false

And this adapter doesn't support _any_ consent legislation? No gpp, tcf, coppa?